tests/v2/validation/prime/prime_test.go

```diff
@@ -4,59 +4,88 @@ import (
 	"testing"
 
 	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters"
+	prime "github.com/rancher/rancher/tests/framework/extensions/prime"
 	"github.com/rancher/rancher/tests/framework/extensions/rancherversion"
 	"github.com/rancher/rancher/tests/framework/pkg/config"
 	"github.com/rancher/rancher/tests/framework/pkg/session"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 	"github.com/stretchr/testify/suite"
 )
 
-const rancherBrand = "suse"
+const (
+	uiBrand         = "ui-brand"
+	defaultRegistry = "system-default-registry"
+	localCluster    = "local"
+)
 
-type PrimeVersionTestSuite struct {
+type PrimeTestSuite struct {
 	suite.Suite
 	session        *session.Session
 	client         *rancher.Client
 	isPrime        bool
 	rancherVersion string
+	primeRegistry  string
 }
 
-func (t *PrimeVersionTestSuite) TearDownSuite() {
+func (t *PrimeTestSuite) TearDownSuite() {
 	t.session.Cleanup()
 }
 
-func (t *PrimeVersionTestSuite) SetupSuite() {
+func (t *PrimeTestSuite) SetupSuite() {
 	testSession := session.NewSession()
 	t.session = testSession
-	primeConfig := new(rancherversion.Config)
 
+	primeConfig := new(rancherversion.Config)
 	config.LoadConfig(rancherversion.ConfigurationFileKey, primeConfig)
+
 	t.isPrime = primeConfig.IsPrime
 	t.rancherVersion = primeConfig.RancherVersion
+	t.primeRegistry = primeConfig.Registry
 
 	client, err := rancher.NewClient("", t.session)
 	assert.NoError(t.T(), err)
+
 	t.client = client
+}
+
+func (t *PrimeTestSuite) TestPrimeUIBrand() {
+	brand, err := t.client.Management.Setting.ByID(uiBrand)
+	if err != nil {
+		t.T().Log(err)
+	}
 
+	prime.CheckUIBrand(t.client, t.isPrime, brand)
```

Comment:
as written, this won't be marked as a failure if the brands don't match if I understand correctly. This will only log the brand, as there's no assertion. Is this what we want?

Reply:
That's an oversight to be honest. I'll fix that to mimic what we do in the provisioning tests.

Reply:
Resolved.